test/fixtures.ts

```diff
@@ -0,0 +1,73 @@
+import type { PathLike } from 'fs';
+import callsite from 'callsite';
+import { DirectoryJSON, IFs, fs, vol } from 'memfs';
+import upath from 'upath';
+
+export class Fixtures {
+  static get(name: string, fixturesRoot = '.'): string {
+    const realFs: IFs = jest.requireActual('fs');
+    return realFs.readFileSync(
+      upath.resolve(Fixtures.getPathToFixtures(fixturesRoot), name),
+      {
+        encoding: 'utf-8',
+      }
+    ) as string;
```

Comment:
```suggestion
    );
```
Is this required for type safety?

Reply:
Yeah, memfs does not have proper overload type for case, when read file function invoked with encoding 'utf-8'

Reply:
You should not use memfs type for real FS. Use the original type